test/e2e/specs/editor/various/shortcut-focus-toolbar.spec.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+const documentToolbarButton = ( page ) => {
+	return page.getByRole( 'button', {
+		name: 'Toggle block inserter',
+		exact: true,
+	} );
+};
+
+const documentToolbarTooltip = ( page ) => {
+	return page.locator( 'text=Toggle block inserter' );
+};
+
+const blockToolbarButton = ( page ) => {
+	return page.getByRole( 'button', { name: 'Paragraph', exact: true } );
+};
+
+const useSelectMode = async ( page ) => {
+	await page.keyboard.press( 'Escape' );
+};
+
+const moveToToolbarShortcut = async ( pageUtils ) => {
+	await pageUtils.pressKeys( 'alt+F10' );
+};
```

Comment:
Even though these are mostly one-liners, I found they made the test cases below much easier to read. They behave as comments when named like this.

Reply:
Ah, nice! Thanks for the tip!